form_utils/fields.py

```diff
@@ -41,9 +41,8 @@ class ClearableFileField(forms.MultiValueField):
     default_file_field_class = forms.FileField
     widget = ClearableFileInput
 
-    def __init__(self, file_field=None, template=None, *args, **kwargs):
```

Comment:
Cannot pass along `file_field` any longer. We aren't using it and it only complicates things in Django 1.11